maths-club-pack/content/facilitator/collatz-conjecture.md

```diff
@@ -0,0 +1,43 @@
+# Collatz Conjecture
+
+## Introduction
+
+The activity is interesting because it is easy to understand the rules and start generating sequences, but it
+is an unsolved problem in mathematics whether all number end up at one. Many people have tried to find
+a sequence that doesn’t go to one, or to prove that all sequences go to one and no one has succeeded.
+There is a prize of at least $2000 dollars if you succeed!  
+
+## Solution
+
+The shortest sequences are going to involve powers of 2, e.g. 1,2,4,8,16,32 etc. as these numbers
+“snowball” to 1. The longest sequence for a number smaller than 100 is for 97, which takes 118 steps but
+finally gets to 1. The number just before, 96, only takes 12 steps!
+
+
+
+## Extension
+
+A computer or tablet would be very useful to try and find very long sequences. Here are some instructions for GeoGebra, which you can find online (https://www.geogebra.org/classic/spreadsheet) or it may be installed on the tablet and you do not need the internet.  
+
+You will need to use the Mod command:  
+
+![](../../images/collatz-conjecture-2.png)
```

Comment:
I thought you would not need an image but I see the markdown disappears when you just copy paste! Really interesting, I don't fully understand why but I think a good solution to use an image instead. The other solution would be to use a code block, which I think will then preserve the characters, e.g.
```
Mod[ <Dividend Number>, <Divisor Number> ]
```